dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/internal/OkHttpUploader.kt

```diff
@@ -91,25 +92,12 @@ internal class OkHttpUploader : Uploader {
     // region Internal
 
     private fun createBody(
-        site: DatadogSite,
         identifier: DdAppIdentifier,
         mappingFile: File,
         repositoryFile: File?,
         repositoryInfo: RepositoryInfo?
     ): MultipartBody {
         val mappingFileBody = mappingFile.asRequestBody(MEDIA_TYPE_TXT)
-        // have to use name, because comparison of enum vs spied/mocked enum directly will always fail
-        val sizeLimit = if (site.name == DatadogSite.US1.name) {
-            MAX_MAP_FILE_SIZE_IN_BYTES_US1
-        } else {
-            MAX_MAP_FILE_SIZE_IN_BYTES
-        }
-        if (mappingFileBody.contentLength() > sizeLimit) {
```

Comment:
Logical and clear change. The only thingy that end user UX is degraded now - I have to wait until the request finishes to find out that it was too big for upload.